backend/question-service/package.json

```diff
@@ -1,8 +1,10 @@
 {
   "name": "question-service",
   "version": "1.0.0",
-  "main": "index.ts",
+  "main": "app.ts",
+  "type": "module",
   "scripts": {
+    "start": "tsx server.ts",
     "dev": "nodemon index.ts",
```

Comment:
i think we need to change this since `index.ts` was renamed

Reply:
i think i'll just remove the dev option then since npm start works